src/js/modules/infragistics.ui.scroll.js

```diff
@@ -2161,42 +2163,72 @@
 				this._switchFromTouchToMixed();
 			}
 
+			/** Get current scroll positions to base the next scroll position from */
+			if (this.options.scrollOnlyHBar) {
+				this._startX = this._getScrollbarHPosition();
+			} else {
+				this._startX = this._getContentPositionX();
+			}
+			if (this.options.scrollOnlyVBar) {
+				this._startY = this._getScrollbarVPosition();
+			} else {
+				this._startY = this._getContentPositionY();
+			}
+
+			/** Get delta for the X axis*/
+			if (evt.wheelDeltaX) {
+				/* Option supported on Chrome, Safari, Opera.
+				/* 120 is default for mousewheel on these browsers. Other values are for trackpads */
+				scrollDeltaX = -evt.wheelDeltaX / 120;
+
+				/** S.K. Fix for Issue #1438 When using trackpad the scroll amount could be less than 1 px, but scrollTop receives only integers, so we need to have it scroll minimum 1 px */
+				if (-minWheelStep < scrollDeltaX && scrollDeltaX < minWheelStep) {
+					scrollDeltaX = Math.sign(scrollDeltaX) * minWheelStep;
+				}
+			} else if (evt.deltaX) {
+				/* For other browsers that don't provide wheelDelta, use the deltaY to determine direction and pass default values. */
+				scrollDeltaX = this._clampAxisCoords(evt.deltaX, -1, 1);
+			}
+
+			/** Get delta for the Y axis*/
 			if (evt.wheelDeltaY) {
 				/* Option supported on Chrome, Safari, Opera.
 				/* 120 is default for mousewheel on these browsers. Other values are for trackpads */
 				scrollDeltaY = -evt.wheelDeltaY / 120;
+
+				/** S.K. Fix for Issue #1438 When using trackpad the scroll amount could be less than 1 px, but scrollTop receives only integers, so we need to have it scroll minimum 1 px */
+				if (-minWheelStep < scrollDeltaY && scrollDeltaY < minWheelStep) {
+					scrollDeltaY = Math.sign(scrollDeltaY) * minWheelStep;
+				}
 			} else if (evt.deltaY) {
 				/* For other browsers that don't provide wheelDelta, use the deltaY to determine direction and pass default values. */
-				scrollDeltaY = evt.deltaY > 0 ? 1 : -1;
+				scrollDeltaY = this._clampAxisCoords(evt.deltaY, -1, 1);
 			}
 
-			if (this.options.smoothing) {
-				//Scroll with small inertia
-				this._smoothWheelScrollY(scrollDeltaY);
+			if (scrollDeltaX) {
+				scrolledX = this._scrollToX(this._startX + scrollDeltaX * scrollStep, true);
```

Comment:
do you need the `scrolledX` assignment as the codacy comment suggests, since it's not used afterwards?